tools/sil-opt/SILOpt.cpp

```diff
@@ -122,6 +122,10 @@ static llvm::cl::opt<bool> EnableOSSAModules(
                    "this is disabled we do not serialize in OSSA "
                    "form when optimizing."));
 
+static llvm::cl::opt<bool> EnableCopyPropagation(
```

Comment:
We actually have two options -enable-copy-propagation and -disable-copy-propagation, both of which default to 'false'. Enabling either one overrides the default pipeline behavior, which depends on the optimization level and whether OSSA is enabled.

Reply:
It looks like swift-frontend has those flags to control the `EnableCopyPropagation` and `DisableCopyPropagation` flags of `the `SILOptions` class sil-opt doesn't.  Considering that they both have independent effects, it seems likely that at some point they'll both be useful for `sil-opt`.  I went ahead and pushed another commit to add it.

The test failures were
```
  Swift(macosx-x86_64) :: Interpreter/SDK/objc_bridge_cast.swift
  Swift(macosx-x86_64) :: Interpreter/SDK/objc_dealloc.swift
```
and seem to be affecting unrelated, affecting other PRs, like https://ci.swift.org/job/swift-PR-macos/28081/consoleText .